WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "2.1.1-beta.2"
+  s.version       = "2.1.1-beta.3"
```

Comment:
I'm not sure how we should best handle this.  We're wanting to incorporate this change into WPiOS 11.9, but it looks like its branched from `develop` which incorporates other changes as well.  I think we'd want to branch off of either `master` (currently at 2.1.0), or the 2.1.0 tag instead. 

The question then is what the version would be, `2.1.0.1 maybe?  Pinging @loremattei and @jkmassel who are wrangling the 11.9 code freeze and release respectively for guidance.

Reply:
I think `2.1.0.1` is the way to go, yes

Reply:
Fair enough. I will close & resubmit.